examples/hal/compute/main.rs

```diff
@@ -4,27 +4,27 @@
 )]
 
 extern crate env_logger;
-extern crate gfx_hal as hal;
 #[cfg(feature = "dx12")]
 extern crate gfx_backend_dx12 as back;
-#[cfg(feature = "vulkan")]
-extern crate gfx_backend_vulkan as back;
 #[cfg(feature = "metal")]
 extern crate gfx_backend_metal as back;
+#[cfg(feature = "vulkan")]
+extern crate gfx_backend_vulkan as back;
+extern crate gfx_hal as hal;
 
 use std::str::FromStr;
 
-use hal::{
-    Backend, Compute, Device, DescriptorPool, Instance, PhysicalDevice, QueueFamily,
-};
-use hal::{queue, pso, memory, buffer, pool, command};
+use hal::{buffer, command, memory, pool, pso, queue};
```

Comment:
is it possible to have `rustfmt` preserve the old style here?

Reply:
What is the old style, exactly?

Reply:
use xxx::{
    YY, ZZZ, WW,
    AA, BB,
};

Reply:
It kind of does that apart from preserving new lines:
https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#imports_layout

Reply:
oh, I see. It split into multiple `use` statements here because one is for modules and another is for types. Otherwise, the "default" makes total sense to me (and I'm pleasantly surprised it is the default).